apps/public/src/app/(general)/samenvatting/Summary.tsx

```diff
@@ -9,17 +9,22 @@ import { SubmitButton, SummaryList } from '@meldingen/ui'
 import { postSummaryForm } from './actions'
 import { BackLink } from '../_components/BackLink'
 
+type GenericSummaryData = {
+  key: string
+  term: string
+  description: string[]
+}
+
 type Props = {
-  data: {
-    key: string
-    term: string
-    description: string[]
-  }[]
+  additionalQuestionsAnswers: GenericSummaryData[]
```

Comment:
This can just be `additionalQuestions` I think. It's both the questions and the answers, no?